assets/js/public.js

```diff
@@ -411,8 +411,8 @@ class Public {
 			}
 			css += '}';
 
-			css  += '@media (hover: none) {';
-			css += `.${hoverBoxClass} { background-image: url('${hoverBgUrl}') !important; } }`;
+			css  += '@media screen and (max-width: 991px) {';
```

Comment:
Spacing with the parenthesis?

Reply:
Per [wordpress coding specification examples](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/css/#media-queries) examples, spacing is not needed